addons/barcodes/__openerp__.py

```diff
@@ -27,10 +29,9 @@
     'depends': ['web'],
     'data': [
         'data/barcodes_data.xml',
-        'barcodes_view.xml',
+        'views/barcodes_view.xml',
         'security/ir.model.access.csv',
-        'views/templates.xml',
+        'views/barcodes_templates.xml',
```

Comment:
Here you can arrange alphabetically.. like
```
A for Apple,
B for Bird,
---
``` 
![alphabet-coloring-pages-printable-apple-and-bird](https://cloud.githubusercontent.com/assets/14940179/12080024/958454b6-b20a-11e5-9b9e-5e8e7a89d071.gif)